selfdrive/car/__init__.py

```diff
@@ -261,6 +261,7 @@ class PlatformConfig:
   platform_str: str
   car_info: CarInfos
   dbc_dict: DbcDict
+  test_routes: list[str] = field(default_factory=list)
```

Comment:
I think we want to keep test routes separate. these should be intrinsic properties of the car

Reply:
kind of sucks that this will be the only thing that doesn't go in here though. maybe we just make really nice error messages around it?

Reply:
there is a nice message when you forget a car `Exception: missing test route for SUBARU IMPREZA LIMITED 2019`, so I suppose that is fine